### PR TITLE
[#2643]: Replace TypeScript with JavaScript in the web demo

### PR DESCRIPTION
**Pull request by hiirrxnn**
### What changes were proposed in this pull request?

Replace TypeScript with JavaScript 

### Why are the changes needed?

for the consistency of the frontend code.

Fix: #2643

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

N/A

### CODE REVIEW

**Comment by hiirrxnn** [comment]
CI bound to fail as configuration file not changed yet. Please review this and let me know if this is how other files have to be changed as well !

**Comment by ch3yne** [comment]
You can see the section `How should we improve?` regarding this issue.

**Comment by hiirrxnn** [comment]
Please review if this looks good.

**Comment by ch3yne** [comment]
Actually, the `web/types` directory can be deleted. The main changes are in the `web/lib/utils/axios` directory.

**Comment by hiirrxnn** [comment]
Please review if the method is correct so that I can continue for rest of the files.

File: web/tsconfig.json
Comment (ch3yne) [inline web/tsconfig.json:1]: Delete this file and create a new file named `jsconfig.json` instead.

File: web/src/types/axios.d.js
Comment (ch3yne) [inline web/src/types/axios.d.js:1]: The `types` directory can be removed.

File: web/src/lib/utils/axios/Axios.js [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+    }, undefined)
+
+    // ** Response result interceptor error capture
+    responseInterceptorsCatch &&
+      isFunction(responseInterceptorsCatch) &&
+      this.axiosInstance.interceptors.response.use(undefined, error => {
+        return responseInterceptorsCatch(axiosInstance, error)
+      })
+  }
+}
```
Comment (ch3yne) [inline web/src/lib/utils/axios/Axios.js:178]: The `}` should not be closed here, as it causes errors.

**Comment by ch3yne** [comment]
> Please review if the method is correct so that I can continue for rest of the files.

I think you can try changing all the mentioned files.

**Comment by jerryshao** [comment]
This is already fixed, thanks @hiirrxnn for your contribution. I'm going to close this.

**Pull request closed without merging**